recipes-utils/imx-misc-tools/files/Add-i.MX8MP-support.patch

```diff
@@ -0,0 +1,253 @@
+From: Alex Gonzalez <alexg@balena.io>
```

Comment:
Did you have to implement this yourself? Sounds weird that the tool doesn't have it.

Reply:
Unfortunately it's not an official NXP tool and the repository was i.MX8MM specific. I looked at other tools like https://github.com/usbarmory/crucible/tree/master which do support a broader range of chipsets, but apart from being in golang their UX was much worse.

In the end finding the OTP differences was just quicker.